sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2172,21 +2172,4 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       }
     }
   }
-
-  test("SPARK-19809 NullPointerException on zero-size ORC file") {
```

Comment:
Should we move the test above `SPARK-22267 Spark SQL incorrectly reads ORC files when column order is different` too? I think basically that's also the same instance.

Reply:
Yep. I'll piggy back here.